src/bp-groups/classes/class-bp-groups-member-suggestions.php

```diff
@@ -155,7 +155,7 @@ public function get_suggestions() {
 		$results = array();
 		foreach ( $user_query->results as $user ) {
 			$result          = new stdClass();
-			$result->ID      = $user->user_nicename;
+			$result->ID      = bp_activity_get_user_mentionname( $user->ID );
```

Comment:
@milanhirapra What about if the activity component is disabled? It will give you the fatal error may be.